src/assets/sass/protocol/components/_sticky-promo.scss

```diff
@@ -41,6 +39,12 @@ $logos: (
     position: fixed;
     width: $content-xs;
     z-index: 10;
+    animation: 600ms ease 1500ms both;
```

Comment:
This delay should only be when it's set to appear automatically. If I show it onclick (or any other event I might use to call the open function) I would expect it to appear right away and having the delay makes it feel sluggish.

I think we can do something like this:
```
&.mzp-js-show-on-load {
    animation-delay: 600ms;
}
```

Reply:
1500ms also feels a little slow... it's a full second and a half, which is an eternity in UI response time. Though it feels ok being slower onload but it's sluggish onclick. So maybe we default to something like 500ms or 750ms, but when it's show-on-load we slow that down to 1500ms or even 1000ms.

Reply:
Revised default `animation-delay` to `300ms` and the show-on-load to `1000ms`.